platform/jewel/ui/src/main/kotlin/org/jetbrains/jewel/ui/component/Menu.kt

```diff
@@ -731,19 +758,20 @@ public fun MenuSubmenuItem(
     submenu: MenuScope.() -> Unit,
     content: @Composable () -> Unit,
 ) {
-    MenuSubmenuItem(canShowIcon, submenu, modifier, enabled, iconKey, interactionSource, style, content)
+    MenuSubmenuItem(canShowIcon, false, submenu, modifier, enabled, iconKey, interactionSource, style) { content() }
 }
 
 @Composable
 internal fun MenuSubmenuItem(
     showIcon: Boolean,
+    isSelected: Boolean,
```

Comment:
Can we call this `selected` instead for consistency?

Reply:
Also, should this value not be passed to the itemState initial creation?

```kotlin
var itemState by
        remember(interactionSource) {
            @Suppress("DEPRECATION") mutableStateOf(MenuItemState.of(selected = selected, enabled = enabled))
        }
```

Reply:
Updated